doc/manual/performance-tips.rst

```diff
@@ -1115,9 +1115,12 @@ should have the following additional properties:
    speedup with :obj:`@simd`.
 
 While :obj:`@simd` needs to be placed in front of a loop, that is not the case
-for :obj:`@inbounds`. It can be placed in front of any expression as long as it
-is inside a function. The only place where it can be outside of a function
-definition is in front of a loop.
+for :obj:`@inbounds`. It can be placed in front of any expression, however
+the compiler isn't always capable of recognizing where to remove bounds check,
```

Comment:
Bit of a run-on. Just end this sentence here and have "Limiting" start a new one.